site/cypress/e2e/Cypress-Feature/test_forum.spec.js

```diff
@@ -46,6 +47,18 @@ const removeThread = (title) => {
     cy.get('.thread-left-cont > .thread-list-item').contains(title).should('not.exist');
 };
 
+const uploadAttachmentAndDelete = (title) => {
+    cy.get('[data-testid="thread-list-item"]').contains(title).click();
+    cy.get('[data-testid="create-post-head"]').should('contain', title);
+    cy.get('[data-testid="edit-post-button"]').first().click();
+    cy.get('[data-testid="upload1"]').find('[for="input-file1"]').selectFile('cypress/fixtures/sea_animals.png');
```

Comment:
Instead of of creating a testid for the div surrounding the element, can you create one for the input its self?

Reply:
` <label for="input-file{{ post_box_id }}" data-testid="input-file{{ post_box_id }}">Upload Attachment</label>` 
i changed like this , its good way ??